<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checkin_Form</class>
 <widget class="QWidget" name="Checkin_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>12</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Recent Check-Ins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="recent_checkins_view"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>12</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Check-In by VID</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter VID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vid_field"/>
      </item>
      <item>
       <widget class="QPushButton" name="checkin_button">
        <property name="text">
         <string>Check-In</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
